src/pdsfile/holdings_maintenance/pds3/update_holdings_for_new_metadata.sh

```
@@ -11,9 +11,14 @@ if [[ $# -ne 2 ]]; then
     exit -1
 fi
 
-HOLDINGS="$(realpath $1)"
+HOLDINGS="$(realpath "$1")"
 VOLSET=$2
 
+if [[ ! "$VOLSET" =~ ^[A-Za-z0-9_]+$ ]]; then
+    echo "Not a volume set name: '$VOLSET'"
+    exit -1
```

Shellcheck (0.11.0)

[error] Can only exit with status 0-255. Other data should be written to stdout/stderr.

(SC2242)